zproject/default_settings.py

```diff
@@ -336,6 +336,18 @@
 # DEFAULT_RATE_LIMITING_RULES.
 RATE_LIMITING_RULES: dict[str, list[tuple[int, int]]] = {}
 
+# Rate limits for endpoints which have absolute limits on how much
+# they can be used in a given time period.
+# These will be extremely rare, and most likely for zilencer endpoints
+# only, so we don't need a nice overriding system for them like we do
+# for RATE_LIMITING_RULES.
+ABSOLUTE_USAGE_LIMITS_BY_ENDPOINT = {
+    "verify_registration_takeover_challenge_ack_endpoint": [
+        # 30 requests per day
+        (86400, 30),
+    ],
+}
```

Comment:
main question is what the limit should be. `30` is a low number but feels generous already - realistically this endpoint should probably average like 1 request per day, with usage spikes never getting anywhere near 30 either.

Reply:
We almost only want to count _failures_ towards the rate limit.

I think this is a fine place to start.  It's not ∞, which is a giant step forward.